pkg/util/system/common_linux.go

```diff
@@ -36,6 +36,24 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// Port2Pid Query pid by port number with the help of lsof command
+func Port2Pid(port int) (int, error) {
```

Comment:
BTW, PortToPid might be a better name. We try to avoid using "2" to replace "to".